Rename import file and class names

juzen2003 opened the pull request:
- Fixes #1334 
- Were any Django, import pipeline, or table_schema files modified? Y
- Were any JavaScript or CSS files modified? N
  - If YES:
    - JSHINT run on all affected files: NA
    - Tested on Chrome, Firefox, Safari, Edge, and iOS: NA
      (Remember to test all browser sizes)
    - Tested for race conditions (with delayed API calls if applicable): NA
- Was the documentation reviewed for necessary changes or additions? NA
  - About
  - Getting Started
  - FAQ
  - API Guide
  - Tooltips
  - Import/database schema

Description of changes:
Rename import files and classes. Here is the summary:
```
Format: Change (original) obs_instrument_{name} to (new) obs_volume_{volume/volset}, and for pds4: obs_bundle_{name}
obs_instrument_cassini_occ (ObsInstrumentCassiniOcc)            -> obs_volume_cassini_occ_common (ObsVolumeCassiniOccCommon)
obs_instrument_cocirs_cube (ObsInstrumentCOCIRSCube)            -> obs_volume_cocirs_01xxx (ObsVolumeCOCIRS01xxx)
obs_instrument_cocirs (ObsInstrumentCOCIRS)                     -> obs_volume_cocirs_56xxx (ObsVolumeCOCIRS56xxx)
obs_instrument_coiss (ObsInstrumentCOISS)                       -> obs_volume_coiss_12xxx (ObsVolumeCOISS12xxx)
obs_instrument_corss_occ (ObsInstrumentCORSSOcc)                -> obs_volume_corss_8xxx (ObsVolumeCORSS8xxx)
obs_instrument_couvis_covims_occ (ObsInstrumentUVISVIMSOcc)     -> obs_volume_couvis_covims_occ_common (ObsVolumeUVISVIMSOccCommon)
obs_instrument_couvis_occ (ObsInstrumentCOUVISOcc)              -> obs_volume_couvis_8xxx (ObsVolumeCOUVIS8xxx)
obs_instrument_couvis (ObsInstrumentCOUVIS)                     -> obs_volume_couvis_0xxx (ObsVolumeCOUVIS0xxx)
obs_instrument_covims_occ (ObsInstrumentCOVIMSOcc)              -> obs_volume_covims_8xxx (ObsVolumeCOVIMS8xxx)
obs_instrument_covims (ObsInstrumentCOVIMS)                     -> obs_volume_covims_0xxx (ObsVolumeCOVIMS0xxx)
obs_instrument_ebrocc (ObsInstrumentEBROCC)                     -> obs_volume_ebrocc_xxxx (ObsVolumeEBROCCXxxx)
obs_instrument_gossi (ObsInstrumentGOSSI)                       -> obs_volume_gossi_0xxx (ObsVolumeGOSSI0xxx)
obs_instrument_hstacs (ObsInstrumentHSTACS)                     -> obs_volume_hstjx_xxxx (ObsVolumeHSTJxXxxx)
obs_instrument_hstnicmos (ObsInstrumentHSTNICMOS)               -> obs_volume_hstnx_xxxx (ObsVolumeHSTNxXxxx)
obs_instrument_hststis (ObsInstrumentHSTSTIS)                   -> obs_volume_hstox_xxxx (ObsVolumeHSTOxXxxx)
obs_instrument_hstwfc3 (ObsInstrumentHSTWFC3)                   -> obs_volume_hstix_xxxx (ObsVolumeHSTIxXxxx)
obs_instrument_hstwfpc2 (ObsInstrumentHSTWFPC2)                 -> obs_volume_hstux_xxxx (ObsVolumeHSTUxXxxx)
obs_instrument_nhlorri (ObsInstrumentNHLORRI)                   -> obs_volume_nhxxlo_xxxx(ObsVolumeNHxxLOXxxx)
obs_instrument_nhmvic (ObsInstrumentNHMVIC)                     -> obs_volume_nhxxmv_xxxx (ObsVolumeNHxxMVXxxx)
obs_instrument_vg28xx_vgiss (ObsInstrumentVG28xxVGISS)          -> obs_volume_vg28xx_vgiss (ObsVolumeVG28xxVGISS)
obs_instrument_vg28xx_vgpps_vguvs (ObsInstrumentVG28xxVGPPS, 
                                   ObsInstrumentVG28xxVGPPSUVS, 
                                   ObsInstrumentVG28xxVGUVS)    -> obs_volume_vg28xx_vgpps_vguvs (ObsVolumeVG28xxVGPPS,          
                                                                                                  ObsVolumeVG28xxVGPPSUVS,
                                                                                                  ObsVolumeVG28xxVGUVS) 
obs_instrument_vg28xx_vgrss (ObsInstrumentVG28xxVGRSS)          -> obs_volume_vg28xx_vgrss (ObsVolumeVG28xxVGRSS)
obs_instrument_vg28xx (ObsInstrumentVG28xx)                     -> obs_volume_vg28xx (ObsVolumeVG28xx)
obs_instrument_vgiss (ObsInstrumentVGISS)                       -> obs_volume_vgiss_5678xxx (ObsVolumeVGISS5678xxx)

Format: Change (original) obs_mission_{name} to (new) obs_volume_{name}_common, and for pds4: obs_bundle_{name}_common
obs_mission_cassini (ObsMissionCassini)                         -> obs_volume_cassini_common (ObsVolumeCassiniCommon)
obs_mission_galileo (ObsMissionGalileo)                         -> obs_volume_galileo_common (ObsVolumeGalileoCommon)
obs_mission_hubble (ObsMissionHubble)                           -> obs_volume_hubble_common (ObsVolumeHubbleCommon)
obs_mission_new_horizons (ObsMissionNewHorizons)                -> obs_volume_new_horizons_common (ObsVolumeNewHorizonsCommon)
obs_mission_voyager (ObsMissionVoyager)                         -> obs_volume_voyager_common (ObsVolumeVoyagerCommon)
```
Known problems:
NA

juzen2003 committed:
Rename import file and class names based on these two formats:
- Format: Change (original) obs_instrument_{name} to
  (new) obs_volume_{volume/volset}, and for pds4: obs_bundle_{name}
- Format: Change (original) obs_mission_{name} to (new)
  obs_volume_{name}_common, and for pds4: obs_bundle_{name}_common

juzen2003 committed:
Clean up code style (indentation) in config_bundle_info.py and
obs_volume_couvis_covims_occ_common.py

rfrenchseti reviewed (changes requested):
A few small changes:
- I think the EBROCC class should be `ObsVolumeEBROCCxxxx` (without the capital X)
- The name of the GOSSI volset is actually `GO_0xxx` so the filename should be `obs_volume_go_0xxx.py` and the class `ObsVolumeGO0xxx`.
- I'm conflicted about the names of the HST classes. `xXxxx` definitely looks odd. I could imagine `01xxxx` or `xxxxx`. What do you think?

juzen2003 committed:
- Change 'ObsVolumeEBROCCXxxx' to 'ObsVolumeEBROCCxxxx'
- Change 'obs_volume_goiss_0xxx' to 'obs_volume_go_0xxx', and
  'ObsVolumeGOISS0xxx' to 'ObsVolumeGO0xxx'
- Change HST classes 'xXxxx' part to 'xxxxx'

juzen2003 committed:
Fixed code style (indentation) in config_bundle_info.py
